third_party/toolchains/backwards_compatibility.bzl

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 The gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_toolchains//rules/exec_properties:exec_properties.bzl", "create_exec_properties_dict", "create_rbe_exec_properties_dict","merge_dicts")
+
+def bc_exec_properties(exec_properties):
+    if True:
+        d = dict(labels = exec_properties[1], **exec_properties[0])
+        return create_rbe_exec_properties_dict(
+            **d
+        )
+    else:
+        labels_dict = {"label:" + key : exec_properties[1][key] for key in exec_properties[1]}
+        return merge_dicts(
+            create_exec_properties_dict(),
+            labels_dict,
+        )
+
+
+def bc_platform(**kwargs):
```

Comment:
do you really need the bc_platform function?

In my PR this seemed to be the same case as `create_rbe_exec_properties_dict`
https://github.com/grpc/grpc/pull/25001/files#diff-e084337b59570d9fb8e3d40828b096dddf9056a9096eb32b44ceeb7fbb0383fcR33